README.md

```diff
@@ -2,15 +2,142 @@
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+- Recognize the benefits of optional arguments
+- Define a method with optional arguments
+- Define a method with optional and required arguments
 
 ## Introduction
 
-## SWBAT 1
+If we revisit the coffee shop scenario, typically a name may be given when you
+place your order so that your name can be called when the order is ready. What
+happens when they don't ask for a name? Usually, they _default_ to calling out
+the order that was placed. This may not be as personalized to you, but it's
+better than not knowing when your order is ready!
+
+In Ruby, we now see that when a method accepts arguments, those arguments become
+required; an error is thrown when those arguments are not passed in on
+invocation of the method. So what do we do when we have optional information?
```

Comment:
"when the method is called"

Reply:
"when we need to tell a method that we might not have the information it expects?"